libs/anglify/src/modules/common/directives/slot-outlet/slot-outlet.directive.ts

```diff
@@ -0,0 +1,29 @@
+import { Directive, EmbeddedViewRef, Input, OnChanges, SimpleChange, TemplateRef, ViewContainerRef } from '@angular/core';
+
+@Directive({
+  selector: '[anglifySlotOutlet]',
+})
+export class SlotOutletDirective implements OnChanges {
+  private viewRef: EmbeddedViewRef<any> | null = null;
+
+  @Input() public anglifySlotOutlet: TemplateRef<any> | undefined;
+  @Input() public anglifySlotOutletContext: Record<string, unknown> | null = null;
+
+  public constructor(private readonly viewContainerRef: ViewContainerRef, private readonly templateRef: TemplateRef<any>) {}
+
+  public ngOnChanges(changes: { anglifySlotOutlet?: SimpleChange; anglifySlotOutletContext?: SimpleChange }): void {
```

Comment:
I'm not sure, but I think "ngOnChanges" will get called infinitely because we are mutating component properties. Can you check that there is no memory leak?

Reply:
I checked this again and for simplicity only added a console.log to the ngOnChanges method. Everything behaved as expected.

By the way: I directly got inspired from the ngTemplateOutlet directive

Reply:
Great.